### PR TITLE
Google sign in feature origin configured

### PR DESCRIPTION
**Pull request by qevyn**

### CODE REVIEW

File: web/selecto/products/templates/products/login.html [hunk 1]
```diff
@@ -17,7 +17,10 @@
     <div class="signin-container">
       <div id="g_id_onload"
          data-client_id='{{ google_client_id }}'
-         data-login_uri="http://localhost:8000/login"
+         data-context="signin"
+         data-ux_mode="popup"
```
Comment (jeffery-do) [inline web/selecto/products/templates/products/login.html:21]: popup?

File: web/selecto/products/templates/products/login.html [hunk 2]
```diff
@@ -17,7 +17,10 @@
...
+         data-login_uri="http://localhost:8000/login/"
```
Comment (jeffery-do) [inline web/selecto/products/templates/products/login.html:22]: probably shouldn't hardcode to localhost. Would need to point that to the current host.

**Review by jeffery-do — APPROVED** [review]
/lgtm

You should pass in the current hostname instead of hardcoding to http://localhost

File: web/selecto/products/templates/products/login.html (hunk 1 above)
Comment (qevyn) [reply web/selecto/products/templates/products/login.html:21]: yea, popup page instead of redirect

**Pull request merged**